js/plugins/ollama/src/index.ts

```diff
@@ -336,6 +203,223 @@ function defineOllamaModel(
   );
 }
 
+async function createOllamaEmbedder(
```

Comment:
what happens to:

https://github.com/firebase/genkit/blob/main/js/plugins/ollama/src/embeddings.ts

I'd expect changes there